src/track.c

```diff
@@ -25,6 +25,17 @@
 
 #define LONG_INTEGRATION_INTERVAL 5
 
+/* Kaplan gives magic parameters for the loop detect algorithm:
+ * DT = 20ms, K1 = 0.0247, K2 = 1.5, Lp = 50, Lo = 240
+ * This corresponds to a low-pass filter time constant of 1.25s
```

Comment:
wow, that seems really slow! I wonder if that's going to miss some brief loss-of-locks